db/db_impl.cc

```diff
@@ -1688,6 +1688,7 @@ Status DBImpl::RecoverLogFiles(const std::vector<uint64_t>& log_numbers,
     }
   }
 
+  bool data_seen = false;
```

Comment:
sorry missed these comments. lmk if you feel strongly about the name. making it a counter is hard because the code is structured to first add all logs to memtables, second check memtables contain data. so by the time memtables are checked we don't know how many logs contributed to them.